### DISCUSSION TITLE
Codes about model deviation

### DISCUSSION THREAD

**Discussion by Daisy315** [deepmodeling/dpgen#1196] [Q&A] [answered]
Hello developers, I would like to ask about the code of calculating the maximum deviation of the model in dpgen? Could you please show me where the specific location is?

**Comment by njzjz** [comment, marked as answer]
Deviation is not calculated within DP-GEN. Instead, it is calculated by DeePMD-kit.

https://github.com/deepmodeling/deepmd-kit/blob/16e91332dc335687d523c22a8a7d74ea74694b6b/source/api_cc/src/DeepPot.cc#L1780

**Reply by Daisy315** [reply]
Many thanks for your reply.
